ansible_collection/hpe/nimble/plugins/modules/hpe_nimble_volume.py

```diff
@@ -354,13 +354,18 @@ def move_volume(
 def update_volume(
         client_obj,
         vol_resp,
+        disassociate,
```

Comment:
Why does this have to be called out specifically?  What's wrong with just checking in the kwargs if the volcoll is set to empty string since that implies a disassociate request.

Reply:
+1

The caller should be able to pass `volcoll=volcoll` as kwargs similar to other attributes.

Reply:
ya. makes sense. will remove disassociate flag